test/prototype_transforms_kernel_infos.py

```diff
@@ -409,15 +434,10 @@ def _compute_affine_matrix(angle, translate, scale, shear, center):
     return true_matrix
 
 
-def reference_affine_bounding_box(bounding_box, *, format, spatial_size, angle, translate, scale, shear, center=None):
-    if center is None:
-        center = [s * 0.5 for s in spatial_size[::-1]]
-
-    def transform(bbox):
-        affine_matrix = _compute_affine_matrix(angle, translate, scale, shear, center)
-        affine_matrix = affine_matrix[:2, :]
+def reference_affine_bounding_box_helper(bounding_box, *, format, affine_matrix):
```

Comment:
Are these changes related to this PR? Is this anything but refactoring?

Reply:
The idea is to use ref affine method to compute expected bboxes for h/v flips.

Reply:
Makes sense!